### DISCUSSION TITLE
How does Offscreen support Chinese character input?

### DISCUSSION THREAD

**Discussion by williamlzw** [cefsharp/CefSharp#4416] [General]
https://github.com/williamlzw/ExDUIR.NETCore/blob/master/ExDUIRTest/TestCustomCtrl.cs
https://github.com/williamlzw/ExDUIR.NETCore/blob/master/ExDUIRTest/CefBrowserWindow.cs
I made a browse box component using cefsharp.offscreen, but I can't input Chinese characters.

**Comment by williamlzw** [comment]
![image](https://user-images.githubusercontent.com/11348670/223028924-d2bb0159-75ba-4ef9-87e8-89dc0e47e987.png)

**Reply by amaitland** [reply]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask questions. Feature request is not to be used for questions.

You can refer to the `cefclient` reference source for an example of implementing `IME`.

https://github.com/chromiumembedded/cef/blob/3e4f8d1b9ccfd720a213a34200b296edb1e475af/tests/cefclient/browser/osr_window_win.cc#L471

If you need help then asking on https://www.magpcss.org/ceforum/index.php would be the place.
